src/pp-annotator/PrzypisAdder.ts

```diff
@@ -22,6 +22,12 @@ interface IPrzypisAdderOptions extends annotator.ui.widget.IWidgetOptions {
 }
 
 export default class PrzypisAdder extends Widget {
+  public static classes = {
```

Comment:
`public` should now be default, at least if you have the recent configuration changes

Reply:
done